src/translations/en.json

```diff
@@ -216,12 +216,13 @@
     "join": "Join"
   },
   "rules": {
-    "rulesTitle": "Rules of Cuttle",
-    "rulesText": "Cuttle is a two player battle card game played with a standard 52-card deck of cards. It has the strategic nuance of trading card games like Magic, with the elegant balance of a standard deck--and you can play it for free! Test your mettle in the deepest card game under the sea!",
+    "cuttleTitle": "What is Cuttle",
+    "cuttleText": "Cuttle is a two player battle card game played with a standard 52-card deck of cards. It has the strategic nuance of trading card games like Magic, with the elegant balance of a standard deck--and you can play it for free! Test your mettle in the deepest card game under the sea!",
```

Comment:
@pradnyabhukan we also need to update the `fr.json` and `es.json` files to ensure that the french and spanish translations for this text also appears in the user's selected language. Please use google translate and then put french and spanish translated versions of this text into the other translation files.

Reply:
I’d actually recommend using chatGPT s it’s usually much closer to a true translation. But definitely compare both to see if there are differences and then ask GPT to explain the nuance if needed.